include/mbedtls/des.h

```diff
@@ -189,6 +193,7 @@ int mbedtls_des_setkey_dec( mbedtls_des_context *ctx, const unsigned char key[MB
  *
  * \return         0
  */
+MBEDTLS_CHECK_RETURN
```

Comment:
Instead of using `MBEDTLS_CHECK_RETURN` directly, we should use a different macro name here, such as `MBEDTLS_CHECK_RETURN_TYPICAL`. `MBEDTLS_CHECK_RETURN` is an abstraction over compiler-specific macros; its semantics is “warn about not checking the result of the following function if the compiler supports it”. `MBEDTLS_CHECK_RETURN_TYPICAL` says “warn about not checking the result if supported and desired for run-of-the-mill functions”. See https://github.com/ARMmbed/mbedtls/issues/3963#issuecomment-876293156.